server/bert_serving/server/helper.py

```diff
@@ -109,6 +109,10 @@ def get_args_parser():
                         help='masking the embedding on [CLS] and [SEP] with zero. \
                         When pooling_strategy is in {CLS_TOKEN, FIRST_TOKEN, SEP_TOKEN, LAST_TOKEN} \
                         then the embedding is preserved, otherwise the embedding is masked to zero before pooling')
+    group2.add_argument('-special_token', action='store_false', default=True,
```

Comment:
better change it to
`'-no_special_token', action='store_true', default=False,`
otherwise, it's strange when user adds `-special_token` but it actually disables the special tokens, which is your current behavior.